_aicspylibczi/Reader.cpp

```diff
@@ -15,8 +16,10 @@ namespace pylibczi {
   {
       m_czireader->Open(std::move(istream_));
       m_statistics = m_czireader->GetStatistics();
+      m_pixelType = libCZI::PixelType::Invalid; // get the pixeltype of the first readable subblock
```

Comment:
It's impossible to have varied pixel types by scene right? I feel like I have personally confirmed this before but just double checking.

Reply:
I'm like 99% sure we have checked this assumption before.... just can't find it in slack.

Reply:
You're going to hate this I know I do. From digging through all the docs I have all I could find is that PixelType has to be consistent across bitmaps which to me says in a "normal" image subblocks could have different PixelTypes but in Mosiac files, they have to be the same for the different M-indexes for the same dims. 

In practice, I can't think how or why one would change PixelType for different parts of an image acquisition. 
I think this is just base file format flexibility that is present but not used or at least not used on pyramid0.